packages/aws-cdk-lib/aws-ecs-patterns/lib/base/network-multiple-target-groups-service-base.ts

```diff
@@ -105,6 +105,15 @@ export interface NetworkMultipleTargetGroupsServiceBaseProps {
    * @default - false
    */
   readonly enableExecuteCommand?: boolean;
+
+  /**
+   * The minimum number of tasks, specified as a percentage of
+   * the Amazon ECS service's DesiredCount value, that must
+   * continue to run and remain healthy during a deployment.
+   *
+   * @default - 0 if daemon, otherwise 50
+   */
+  readonly minHealthyPercent?: number;
```

Comment:
could you add an `@see` here with a link to the docs?

Reply:
also, I think I'd prefer this to be `minimumHealthyPercent` to match CloudFormation.

Reply:
@cheruvian I didn't see it in `BaseServiceOptions`. Apologies, you can change it to make that.

Reply:
Rest LGTM, if you want to change this back.

Reply:
Any thoughts on whether this should actually just `extend Omit<BaseServiceOptions, ...>`?

Reply:
I think this is fine but we'll see what the maintainer's say.